packages/teuchos/numerics/src/Teuchos_ExpandScalarTypeMacros.hpp

```diff
@@ -43,59 +43,45 @@
 #define TEUCHOS_EXPAND_SCALAR_TYPE_MACROS_HPP
 
 
-/*! \file Teuchos_UnitTestHelpers.hpp
+/*! \file Teuchos_ExpandScaleTypeMacros.hpp
 
 \brief Macros for expanding code
 */
 
 
-#include "Teuchos_ConfigDefs.hpp"
+#include "Teuchos_ExplicitInstantiationHelpers.hpp"
 
-#ifdef HAVE_TEUCHOS_FLOAT
+#ifdef HAVE_TEUCHOS_INST_FLOAT
 #  define TEUCHOS_MACRO_EXPAND_FLOAT(INSTANT_MACRO)\
     INSTANT_MACRO(float)
-#  define TEUCHOS_CLASS_TEMPLATE_INSTANT_FLOAT(CLASSNAME)\
-    template class CLASSNAME<float>;
 #else
 #  define TEUCHOS_MACRO_EXPAND_FLOAT(INSTANT_MACRO)
-#  define TEUCHOS_CLASS_TEMPLATE_INSTANT_FLOAT(CLASSNAME)
 #endif
 
 
 #define TEUCHOS_MACRO_EXPAND_DOUBLE(INSTANT_MACRO)\
   INSTANT_MACRO(double)
-#define TEUCHOS_CLASS_TEMPLATE_INSTANT_DOUBLE(CLASSNAME)\
-  template class CLASSNAME<double>;
 
-#if defined(HAVE_TEUCHOS_COMPLEX) && defined(HAVE_TEUCHOS_FLOAT)
+#ifdef HAVE_TEUCHOS_INT_COMPLEX_FLAOT
```

Comment:
Ditto

Reply:
Ditto

Reply:
Resolved in rebased commit https://github.com/trilinos/Trilinos/pull/10720/commits/cd5ee93bb53f82daffe2b31e1fbce051d4433397